### PR TITLE
Data source init properties fix

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -70,8 +66,16 @@ rp:
...
+    type: com.zaxxer.hikari.HikariDataSource
+    driverClassName: org.postgresql.Driver
+    jdbcUrl: ${rp.db.url}
+    username: ${rp.db.user}
+    password: ${rp.db.pass}
+    maximumPoolSize: 27
+
   db:
-    url: jdbc:postgresql://${rp.db.host}:${rp.db.port}/${rp.db.name}?user=${rp.db.user}&password=${rp.db.pass}
+    url: jdbc:postgresql://${rp.db.host}:${rp.db.port}/${rp.db.name}
```
Comment (pbortnik) [inline src/main/resources/application.yaml]: why did you remove user and password from here?

Comment (Yumfriez) [reply src/main/resources/application.yaml]: This statement was split into 3 key-value statements:
url: url
username: username
password: password
